### PR TITLE
Setup travis ci

### PR DESCRIPTION
**Pull request by mcolombo87**
### Summary
Travis CI it's ready for be use it in this project and Readme file was modified to contain the build status of master branch.
### ScreenShots
image
image
### Trello
https://trello.com/c/Wn3vSCFB/13-configurar-travisci
### Known Issues/Notes
The build is failing because the only single step in scripts is run rubocop, but rubocop is waiting for jump in (see [PR#2](https://github.com/wolox-training/mec-rails/pull/2)).

### CODE REVIEW

**Comment by floresv** [comment]
Review build of `travis`, It's failing.

**Comment by mcolombo87** [comment]
> Review build of `travis`, It's failing.

Yes I know. That's it because the build attempt execute rubocop but it's not available yet (PR#2). I can fix it removing that line or wait for the Pull Request merge.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  - bundle exec rubocop app spec -R --format simple
+#  - bundle exec scss-lint app/assets/stylesheets/
+
+#notifications:
+#  email:
+#    recipients:
+#      - rails-bootstrap-dev@wolox.com.ar
+
+cache:
+  bundler: true
```
Comment (facucompe) [inline .travis.yml:43]: That red symbol indicates that you should leave an empty line at the bottom.  It's just part of my OCD (TOC).

**Review by facucompe — DISMISSED** [review]

**Review by floresv — DISMISSED** [review]

File: .travis.yml (hunk 1 above)
Comment (mcolombo87) [reply .travis.yml:43]: :+1:

**Review by facucompe — APPROVED** [review]

**Review by floresv — APPROVED** [review]

**Pull request merged**